### ISSUE TITLE
Problems with training and inference on custom dataset

### ISSUE THREAD

**Issue by pfuerste** [matterport/Mask_RCNN#2602]
I am training on my own dataset, the loss is decreasing, but I have some problems, which are because I do not know how to use some parameters correctly. This is my training Configuration:
`IMAGES_PER_GPU = 8

    # Number of classes (including background)
    NUM_CLASSES = 1 + 2  # Background + 2 Object classes

    # Number of training steps per epoch
    STEPS_PER_EPOCH = 50

    # Skip detections with < 90% confidence
    DETECTION_MIN_CONFIDENCE = 0.9

    EPOCHS = 50

    VALIDATION_STEPS = 10

    MAX_GT_INSTANCES = 70

1. The training performance is not as good as I'd hoped. I trained for 50 Epochs now, loss is at ~0.8, but that my be because of the size of my data set, it is only ~350 images yet. I also only trained the heads yet. Still, the predicted images do not look like the model is working, but the other losses are low.
 
![grafik](https://user-images.githubusercontent.com/26963125/122377476-68b0dd00-cf65-11eb-8e6e-aaa8723fcede.png)

2. As you see, I did not change IMAGE_SHAPE, IMAGE_Min_DIM or IMAGE_MAX_DIM in the config. When loading the model in inference mode and not setting those parameters, it throws
`Exception: Image size must be dividable by 2 at least 6 times to avoid fractions when downscaling and upscaling.For example, use 256, 320, 384, 448, 512, ... etc.`
(I also set `GPU_COUNT = 1` and `IMAGES_PER_GPU = 1` for inference.)
The training data is 250x420x4 Pixel (TIF, last one is depth to be ignored here), as is the data I am trying to predict on. 
When I set
`config.IMAGE_SHAPE = [256, 448, 3]
config.IMAGE_MIN_DIM = 256
config.IMAGE_MAX_DIM = 448
` for inference, the model loads, but I get outputs like this one:
![grafik](https://user-images.githubusercontent.com/26963125/122378585-5f744000-cf66-11eb-8a14-b2c3936c2466.png)

How do I set the shape parameters so they are right? This prediction is not just a case of bad performance, right?
